src/FilterControls/SortOption.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import './SortOption.less';
+
+const handleClick = function(active, selectHandler, unselectHandler) {
+	return !active ? selectHandler: unselectHandler;
```

Comment:
having to do pass down a prop and two event handlers was awkward, but let me keep this component pure and stateless. I'd love to hear alternative patterns

Reply:
A couple examples of the above:
 - [Connecting props & actions via Redux](https://github.com/TroyAlford/axis-wiki/blob/a92f0b96f34f5155d8fa876c1ac94c91c49b2881/source/components/FacebookWrapper.js)
 - [Passing MST collection and sorting on the fly](https://github.com/TroyAlford/sheetforge/blob/develop/source/components/SkillSection.js)